Misc/NEWS.d/next/Build/2019-01-02-11-23-33.bpo-35642.pjkhJe.rst

```diff
@@ -0,0 +1 @@
+Remove asynciomodule.c from pythoncore.vcxproj
```

Comment:
please mention somehow that it's already built by "_asyncio.vcxproj (providing _asyncio.pyd)".